src/analyzing/Constant_propagation.ml

```diff
@@ -814,14 +814,18 @@ let propagate_dataflow lang ast =
       let xs =
         AST_to_IL.stmt lang (G.Block (Tok.unsafe_fake_bracket ast) |> G.s)
       in
+      (* No implicit return analysis needed for top-level scope,
```

Comment:
Why do we have this comment about implicit-return in a module that only concerns constant propagation?

Reply:
OK I see now, I think it's fine to have these comments, but I would put all the info in `CFG_build.mli` and then here just have a very short comment, one can always go to `CFG_build.mli` to know more.

Reply:
Sounds good! I'll make the change.